packages/components/select-menu/src/select-menu.js

```diff
@@ -21,6 +21,9 @@ export class TSSelectMenu extends TSElement {
 			items: { type: Array, reflect: true },
 			/** Allow users to select several options or not. */
 			multiselect: { type: Boolean, reflect: true },
+			/** Do not show the apply button and directly rise select-menu-changed when the selection changes.
+			 * Only affects the behaviour when multiselect is enabled, for single selection this is the default behavior. */
+			noApplyButton: { type: Boolean, reflect: true },
```

Comment:
Please add the `attribute` property for it. 
attribute: 'no-apply-button'

I'm not a fan of the name but it's really hard to find any better name for property of this feature. `autoSelect` `disableApplySelect`, .... I don't think these are any better, but maybe you had some other options in mind to share?

Reply:
not very happy with the name also, but do not have any other great ideas, 
hideApplyPanel
noApplyPanel
doNotShowApply
also no hard opinion on it

Reply:
I think the one that you chose is most clear option for now. Hopefully we can kill the apply button functionality all together in next iterations! 
But please do add the attribute for it to be kebab-case.